thrimshim/thrimshim/main.py

```diff
@@ -94,6 +125,13 @@ def update_row(ident, new_row):
 	for extra in extras:
 		del new_row[extra]
 
+	#validate title length - YouTube titles are limited to 100 characters, 9 reserved for "DBXXXX - ".
```

Comment:
Hm. So I guess thrimbletrimmer is assuming that the cutter is automatically prepending that?
I'd rather avoid hard-coding those kinds of specifics in the cutter - maybe we should do that here instead? ie. write `DBXXXX - <user's input>` as the title value. Or more generally any arbitrary prefix.

On a related note, we also need to set tags, probably in the same place - either in the cutter, or here. Or maybe even in sheet sync. Let's discuss that further. This code is fine for now.

Reply:
We had the cutter handle that in old Wubloader; so I'd just copied that forward. Makes perfect sense to just move the default title prefix into Thrimbletrimmer though; and keeps this section more generic. For now I'll switch to the 100 characters.

Previously all tags were set by the cutter automatically, since it was just a standard set of tags (plus the category). Maybe have Sheetsync set the default tags in an extra column, the Thrimbletrimmer can compile the final set (and possibly expose them to the editor), and submit them back?

Reply:
Hm. Actually now I think about it, it probably makes sense to make both the title prefix (`DBXXXX - `) and tags be defined as part of the "upload location" config in the cutter, since it's youtube specific, and even more than that it's "main channel" specific. We don't want to do it for things we might want to upload to an alternate channel. Public vs unlisted falls under the same category.